{{cookiecutter.project_name}}/.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+    - name: Scverse Community Forum
+      url: https://discourse.scverse.org/
+      about: If you have questions about “How to do X”, please ask them here.
```

Comment:
I think this needs a little more consideration. What we're currently doing for discourse is creating tags for each package. That's a manual process, and one that probably shouldn't be automate on package instantiation.

See current organization for discourse in this comment and the linked google doc: https://github.com/scverse/governance/issues/26#issuecomment-1150123357

tbh, I think using issues for discussion is probably fine for the first little while.

Reply:
It would be a nice way to advertise discourse as *the* place for single-cell analysis discussions, though. 
We could also have a tag for "other package" and start creating dedicated tags once the traffic increases.

Reply:
@adamgayoso, what do you think?

Reply:
yes everyone should use it

Reply:
I agree and tbh I would be very strict to what can be reported as github issue: only bug report and feature request, everything else should be discourse IMHO. How do we do that unclear, as in we can't not answer issues.